src/template_feed/Microsoft.DotNet.Common.ItemTemplates/Microsoft.DotNet.Common.ItemTemplates.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
should it be in the root folder instead? e.g. `/template_feed` instead of `/src/template_feed`?

Reply:
As we discussed before, it's /src/template_feed. Putting it in root folder makes more sense since it's not part of SDK. I will make this change.